CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## Unreleased
 
+### Changed
```

Comment:
@michaeltlombardi keep me honest with the changlog format

Reply:
These all work for the `Changed` section, but the changelog here doesn't specify the new features or bug fixes - this is user facing, so anything the end-user of the editor services would notice (particularly new features) should probably be included here, even if the actual work happened in a related project we consume.

I'm not sure about this long term, but right now this extension drives the development of those other independent projects.

Reply:
I first tried this out by copying the changelog from puppet-editor-services here, but that didn't work so well. Since we use markdown links, it looked just like puppet-vscode issues but the puppet-editor-services numbering was lower than puppet-vscode so the numbers looked off. I then thought to add puppet-editor-services somewhere in the display part, but that meant I would have to change everything to match or else it would look weird.

So, I put up links to the changelog in each line instead that referred to a different project